modules/couchbase/src/test/java/org/testcontainers/couchbase/CouchbaseContainerTest.java

```diff
@@ -61,13 +64,36 @@ public void testBasicContainerUsage() {
         }
     }
 
+    @Test
+    public void testBasicContainerUsageForCommunityContainer() {
```

Comment:
I wonder why this new method is specifically necessary for the community container. Can't we make the original `testBasicContainerUsage` a parameterized test?

Reply:
We could, but I don't know how to write parameterized tests in junit4 for only one test method.

Reply:
You are right, for this we would need to extract to its own class, so maybe not worth it.
However, why does the test itself differ from the `testBasicContainerUsageForEnterpriseContainer` body? Is it important to test other things explicitly for community container?

Reply:
The only reason behind both tests is to check if the containers can be started and some smoke tests. Without my changes the Community edition failed to start.

Reply:
I was referring to the code as part of `setUpClient` in `testBasicContainerUsageForCommunityContainer`. This is not present in the `testBasicContainerUsageForEnterpriseContainer` test and I like to understand why the test differs in this way.

Reply:
I am sorry, I don't get it. The tests of Community and Enterprise are exactly (and should be) the same except the different docker images.

Reply:
Sorry you are right, I misread the diff.